deploy/docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Docker image for the primary terria map application server
-FROM node:16
+FROM node:16-slim
```

Comment:
This Dockerfile runs the node process as `root` unlike the Dockerfile in the root of this repository that runs it as `node`.